homework/index.js

```diff
@@ -30,14 +30,120 @@
     return elem;
   }
 
+  function clearContainer(container) {
+    while (container.firstChild) {
+      container.removeChild(container.firstChild);
+    }
+  }
+
+  function renderError(err) {
+    const listContainer = document.getElementById('list-container');
+    clearContainer(listContainer);
+    const root = document.getElementById('root');
+    createAndAppend('div', root, { text: err.message, class: 'alert alert-error' });
+  }
+
+  function addRow(tbody, label, repository) {
+    const tr = createAndAppend('tr', tbody);
+    createAndAppend('td', tr, { text: `${label}:`, class: 'label' });
+    if (label === 'Repository:') {
+      const urlRepo = repository.html_url;
+      const td = createAndAppend('td', tr);
+      createAndAppend('a', td, {
+        text: repository.name,
+        href: urlRepo,
+        target: '_blank',
+      });
+    }
+    if (label === 'Description:') {
+      createAndAppend('td', tr, { text: repository.description });
+      console.log(repository.description);
+    }
+    if (label === 'Forks:') {
+      createAndAppend('td', tr, { text: repository.forks });
+    }
+
+    if (label === 'Updated:') {
+      createAndAppend('td', tr, { text: repository.updated_at });
+    }
+  }
+
+  function createInfoList(repository, container) {
+    clearContainer(container);
+    const div = createAndAppend('div', container, { id: 'info_div' });
```

Comment:
You are generating some invalid HTML here. First of all, you don't need a `ul` here as you are not creating a list. Moreover, a `ul` can only have `li` elements as its children, and not a `div` as you have here.

Inside your `tbody` you also have two empty `tr` elements which serve no purpose at all:

```html
<ul id="list-container">
  <div id="info_div">
    <table>
      <tbody>
        <tr>
          <td class="label">Repository::</td>
          <td><a href="https://github.com/HackYourFuture/alumni" target="_blank">alumni</a></td>
        </tr>
        <tr>
          <td class="label">Description::</td>
          <td>Alumni Guides, everything you need to land a job</td>
        </tr>
        <tr>
          <td class="label">Forks::</td>
          <td>9</td>
        </tr>
        <tr>
          <td class="label">Updated::</td>
          <td>2019-06-21T11:38:21Z</td>
        </tr>
      </tbody>
      <tr></tr>
      <tr></tr>
    </table>
  </div>
</ul>
```